src/vendor/github.com/radondb/shift/shift/dump_progress.go

```diff
@@ -0,0 +1,206 @@
+package shift
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/radondb/shift/xbase"
+)
+
+const (
+	metaDir = "/tmp/radon-meta-shift"
+)
+
+// ShiftProgress used to store progress rate msg during shift data
+type ShiftProgress struct {
+	DumpProgressRate string `json:"dump-progress-rate"`
+	DumpRemainTime   string `json:"remain-time"`
```

Comment:
`DumpRemainTime ` means the remain time for dump?  How is this parameter value calculated?  Is there any value other than 0? How about replace with migration duration?